kits/news-digest/prompts/news-digest_llmnode-683_system_0.md

```
@@ -0,0 +1,7 @@
+You are an information extraction assistant. Extract genuine news articles from scraped webpage content.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)